cli.js

```diff
@@ -199,16 +199,18 @@ function statistics(objects) {
     countAmount = 0,
     countDate = 0;
   for (var key in objects) {
-    var object = objects[key];
-    if (!object.error) {
-      if (object.amount && object.date) {
-        ++countTotal;
-      }
-      if (object.date) {
-        ++countDate;
-      }
-      if (object.amount) {
-        ++countAmount;
+    if ({}.hasOwnProperty.call(key, objects)){
```

Comment:
code climate advises us that we should be careful when using objects as maps in JS http://www.2ality.com/2012/01/objects-as-maps.html @danschultzer any comment on this